### ISSUE TITLE
golden sample images are blocky

### ISSUE THREAD

**Issue by joeblew99** [neohelden/Flutter-AdaptiveCards#15]
![sample14-base](https://user-images.githubusercontent.com/11999859/59562450-2fb79800-902d-11e9-8438-f098c9b8c20c.png)

Maybe something to do with fonts ?

x-MacBook-Pro:cards apple$ flutter doctor -v
[✓] Flutter (Channel master, v1.7.5-pre.12, on Mac OS X 10.14.5 18F132, locale en-DE)
    • Flutter version 1.7.5-pre.12 at /Users/apple/fvm/master
    • Framework revision 07dc236237 (3 hours ago), 2019-06-16 19:18:39 +1200
    • Engine revision f44b7b5662
    • Dart version 2.4.0

[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/apple/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/apple/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/apple/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit
      https://flutter.dev/setup/#android-setup for detailed instructions.

[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2.1, Build version 10E1001
    • CocoaPods version 1.7.1

[✓] iOS tools - develop for iOS devices
    • ios-deploy 1.9.4

[!] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.35.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.1.0

[✓] Connected device (1 available)
    • Nexus 5X • 024e140e5e46c929 • android-arm64 • Android 8.1.0 (API 27)

**Comment by Norbert515** [comment]
Fonts are pretty weird in Flutter Golden tests (that's also the main reason why I haven't included more).

I added the robot font as showing in this [issue](https://github.com/flutter/flutter/issues/17700) but I still get those blocks for most of the text.

I'll get back to this issue once more progress has been made in that area.

**Comment by tvolkert** [comment]
@gamebox didn't you get real fonts working in `flutter_tester`?

**Comment by gamebox** [comment]
@tvolkert Yes, but the Fonts must be loaded into `FontLoader` first.  At least at the time, the AssetBundle wasn't available for the flutter_test environment.

**Comment by gamebox** [comment]
https://api.flutter.dev/flutter/services/FontLoader-class.html

This requires loading the bytes of the TTF font and using the above class to register the font with the engine.
